egs/wsj/s5/steps/nnet2/get_egs_discriminative2.sh

```diff
@@ -256,39 +255,23 @@ if [ $stage -le 10 ]; then
 priors_egs_list=
 for y in `seq $num_archives_priors`; do
```

Comment:
It's OK-- creating the egs for the priors is fast but doing the computation
using them is not that fast.  So I didn't propose to have just one archive
of priors.
Dan


On Thu, Oct 29, 2015 at 4:49 PM, Vijayaditya Peddinti <
notifications@github.com> wrote:

> In egs/wsj/s5/steps/nnet2/get_egs_discriminative2.sh
> <https://github.com/kaldi-asr/kaldi/pull/305#discussion_r43442325>:
>
> > @@ -256,39 +255,23 @@ if [ $stage -le 10 ]; then
> >  priors_egs_list=
> >  for y in `seq $num_archives_priors`; do
>
> Why is num_archives_priors still being used ? I thought the entire point
> was to just have one job for prior computation.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kaldi-asr/kaldi/pull/305/files#r43442325>.
>